tests/unit/type/hash.tcl

```diff
@@ -589,15 +589,15 @@ start_server {tags {"hash"}} {
         catch {r hincrbyfloat smallhash str 1} smallerr
         catch {r hincrbyfloat bighash str 1} bigerr
         set rv {}
-        lappend rv [string match "ERR*not*float*" $smallerr]
-        lappend rv [string match "ERR*not*float*" $bigerr]
+        lappend rv [string match "ERR *not*float*" $smallerr]
+        lappend rv [string match "ERR *not*float*" $bigerr]
```

Comment:
I'm not certain I would bother to change all the places that checked the error without space. 
It is indeed slightly better, but adds unnecessary changes to many places. 
Maybe the main advantage is that if these lines will get copied, people will get the right thing. 

What's bothering me more, is why don't we see some similar fix in scripting.tcl. That's the one that missed the issue you're fixing.

Reply:
It's because I forgot to commit the second half of the changes. I only captured the `"ERR*` and not `{ERR*`.

I'm going to posit it's still worth doing, since people are going to copy from these. There are no other mistakes, but I think it's good hygiene here.